### ISSUE TITLE
AppleSkin 2.0.0+ incompatible with Sandwichable

### ISSUE THREAD

**Issue by ghost** [squeek502/AppleSkin#128]
Crash at start with Sandwichable mod.
Version: appleskin-fabric-mc1.16.x-2.1.0 + sandwichable-1.2-beta5--1.16.5
Logs: https://pastebin.com/9qUY1Rg6

**Comment by squeek502** [comment]
Looks similar to the incompatibility in https://github.com/squeek502/AppleSkin/issues/115. Sandwichable will probably want to use the AppleSkin API instead of applying a mixin to AppleSkin classes:

https://github.com/squeek502/AppleSkin/tree/1.16-fabric#for-mod-developers

**Comment by FoundationGames** [comment]
Yes, sandwichable was originally adding compatibility before the new API was added, and the old "API" consisted of an interface to implement over item classes, preventing me from reasonably making appleskin an optional dependency.
